src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier supplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        return new Ball(supplier.getRandomColor(), random.nextInt(100));
```

Comment:
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery)
**All magic numbers in your code should be constants.**
Make 100 constant.